apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -280,20 +286,37 @@ const AttestationsTable = ({
                   </Tr>
                 ))}
               </Thead>
-              <Tbody {...getTableBodyProps()}>
-                {rows.map(row => {
-                  prepareRow(row);
-                  return (
-                    <Tr {...row.getRowProps()}>
-                      {row.cells.map(cell => (
-                        <Td {...cell.getCellProps()} borderColor="gray.100">
-                          {cell.render('Cell')}
-                        </Td>
-                      ))}
-                    </Tr>
-                  );
-                })}
-              </Tbody>
+              <InfiniteScroll
```

Comment:
This should wrap the table I am seeing a validate dom nesting error in the console

Reply:
And same in the attestation table